features/upgrade/cloudcredential/upgrade.feature

```diff
@@ -0,0 +1,23 @@
+Feature: CloudCredentialOperator components upgrade tests
+  @upgrade-prepare
+  @admin
+  # @author lwan@redhat.com
+  # @case_id OCP-34260
```

Comment:
case ID should be with upgrade check scenario.

Reply:
> case ID should be with upgrade check scenario.

Thanks for reviewing, has updated, is this what you mean?